Update _index.md - GraphQL Introspection endpoint related note. (#6090)
Kong/docs.konghq.com: app/_hub/kong-inc/graphql-rate-limiting-advanced/overview/_index.md

  * Redis configuration values are ignored if the `cluster` strategy is used.
  * PostgreSQL 9.5+ is required when using the `cluster` strategy with `postgres` as the backing Kong cluster datastore.
  * The `dictionary_name` directive was added to prevent the usage of the `kong` shared dictionary, which could lead to `no memory` errors.

>   * Introspection endpoint is generated based on the Kong service path, hence service path should be defined with actual path instead of appending from the route path.

Reviewer: ```suggestion
  * The introspection endpoint is generated based on the Kong service path, so the service path should be defined with an actual path instead of appending from the route path.
```
Using "so" instead of "hence" for American english phrasing (we use American english as the basis for our docs).

>     - A known limitation is we cannot have separate paths for query and introspection endpoints.

Reviewer: ```suggestion
    * Known limitation: The query and introspection endpoints _cannot_ have separate paths.
```

    - Example: While using KIC, if the query and introspection endpoints are at path /graphql, should be configured like this

>       - Add `konghq.com/strip-path: "true"` annotation to the ingress resource

Reviewer: ```suggestion
      * Add the `konghq.com/strip-path: "true"` annotation to the ingress resource
```
